CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Release Notes
 
+## Version 1.23.0 - 2021-09-03
+
+Features:
+* Add `--extra-disks` capability to kvm2 driver [#12351](https://github.com/kubernetes/minikube/pull/12351)
+* Support Rootless Docker [#12359](https://github.com/kubernetes/minikube/pull/12359)
+* Add support for tcsh in docker-env subcommand [#12332](https://github.com/kubernetes/minikube/pull/12332)
+* Support Ingress on MacOS, driver docker [#12089](https://github.com/kubernetes/minikube/pull/12089)
+* KVM2 driver for linux/aarch64 [#11768](https://github.com/kubernetes/minikube/pull/11768)
```

Comment:
lets remove this one, since ISO is not ready it could be misleading

Reply:
removed